/home/phil/usr243_Fc3/bin/python simpletest.py

--send arrays from C++ to python: --

{'myDoubles': array([ 1.,  2.,  3.,  4.,  5.,  6.]), 'myInts': array([ 88,  99, 100])}

--send array from python to C++:-- 


data values on c++ side: 
1
2
3
4

--send the wrong type --

Traceback (most recent call last):
  File "simpletest.py", line 19, in ?
    simple_ext.testFromPython(theArray)
TypeError: expected Numeric type PyArray_DOUBLE, found Numeric type PyArray_LONG

--send the wrong rank --

Traceback (most recent call last):
  File "simpletest.py", line 26, in ?
    simple_ext.testFromPython(theArray)
RuntimeError: expected rank 2, found rank 1

--print docstrings --


module rcsid:$Id$
 
docstring for module

Doc string for testToPython

Doc string for testFromPython

Doc string for testFromPython2

numpy version:  1.0.2.dev2885
~/philprojects/pythonlibs/num_util/num_util phil@gull% 
